.github/workflows/release-chart.yaml

```diff
@@ -1,16 +1,35 @@
 name: Release Helm Chart
 on:
-  push:
+  pull_request_target:
+    types:
+      - closed
     branches:
       - develop
 
 jobs:
-  release-chart:
+  release-charts:
+    if: github.event.pull_request.merged == true
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Publish Mayastor Helm chart
-        uses: stefanprodan/helm-gh-pages@v1.5.0
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          charts_dir: .
+    - uses: actions/checkout@v3
+      with:
+       fetch-depth: 0
+    - name: Chart Changed
+      id: chart_changed
+      env:
+        GITHUB_BRANCH: ${{ github.head_ref || github.ref_name }}
+      run: |
+        if ! git diff --exit-code ${{ github.event.pull_request.head.sha }} ${{ github.event.pull_request.base.sha }} ./chart; then
```

Comment:
This can be done a bit more simply running this workflow on all push events on `develop`.
ref1: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore
ref2: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
Ah good shout, done!